src/components/enter-email/enter-email-controller.ts

```diff
@@ -228,3 +231,23 @@ function handleBadRequest(
   const error = formatValidationError("email", req.t(errorMessageKey));
   return renderBadRequest(res, req, RE_ENTER_EMAIL_TEMPLATE, error);
 }
+
+function setUpLocks(req: any, lockoutArray: LockoutInformation[]) {
+  lockoutArray.forEach(function (lockoutInformation) {
+    if (lockoutInformation.lockType == "codeBlock") {
+      const lockTime = (req.session.user.wrongCodeEnteredLock = new Date(
```

Comment:
There's a helper function you can use for this now in the lockout helper I introduced - `timestampNMinutesFromNow`